### PR TITLE
Add OpenMRS Sync 2.0 to indexing

### PR DESCRIPTION
**Pull request by tomaszmueller**
This commit is to make AddOns index the OpenMRS Sync 2.0 module

### CODE REVIEW

File: src/main/resources/add-ons-to-index.json [hunk 1]
```diff
@@ -104,6 +104,26 @@
...
+    {
+    "uid": "org.openmrs.module.sync2",
+    "type": "OMOD",
+    "name": "Sync 2.0",
+    "tags": [
+        "sync"
+    ],
+    "maintainers": [
+      {
+        "name": "Tomasz Mueller",
```
Comment (dkayiwa) [inline src/main/resources/add-ons-to-index.json]: Shouldn't this be? https://github.com/openmrs/openmrs-module-sync2

**Comment by reubenvarghese1** [comment]
Hi @tomaszmueller Thanks for the PR. Since your module is an OMOD, can you please move your code such that the module is alphabetically in the right place  with the other omods?

**Comment by tomaszmueller** [comment]
@reubenvarghese1 Done

File: src/main/resources/add-ons-to-index.json [hunk 2]
```diff
@@ -3233,6 +3213,23 @@
...
+    "name": "Sync 2.0",
+    "maintainers": [
+      {
+        "name": "Tomasz Mueller",
+        "url": "https://github.com/tomaszmueller"
+      }
+    ],
+    "backend": "org.openmrs.addonindex.backend.Bintray",
+    "bintrayPackageDetails": {
+      "owner": "tomaszmueller",
```
Comment (reubenvarghese1) [inline src/main/resources/add-ons-to-index.json]: Hi @tomaszmueller , on further inspection, I realised that the Bintray details may not be accurate. Could you re-check that for me please? 
I believe that https://bintray.com/openmrs/omod/sync2 is the location of the module?
If yes, then the owner is openmrs

File: src/main/resources/add-ons-to-index.json [hunk 3]
```diff
@@ -3233,6 +3213,23 @@
...
+      "repo": "omod",
+      "package": "openmrs-module-sync2"
```
Comment (reubenvarghese1) [inline src/main/resources/add-ons-to-index.json]: Similarly, the package is sync2

**Comment by tomaszmueller** [comment]
OK, @reubenvarghese1 changes applied. Anything else?

**Comment by reubenvarghese1** [comment]
@tomaszmueller  Looks good to me, thanks!

**Comment by djazayeri** [comment]
@tomaszmueller this isn't exactly the right place for this conversation, but can I suggest that you should change the naming you're using here to just "Sync 2" and not "Sync 2.0"? Because there will be multiple releases, 2.0, 2.1, 2.2, etc. And it would be pretty confusing to say "Sync 2.0, version 2.1".

**Pull request merged**